iiasa/message-ix-models
# Add MESSAGEix-Materials module

This PR migrates .model.material (i.e. MESSAGEix-Materials) from the (private) message_data repo to this message-ix-models package, following the procedure described in https://github.com/iiasa/message-ix-models/pull/89.

## How to review

- Read the diff and note that the CI checks all pass.
- Run the build, solve, and report commands as described in the documentation. 
- Build the documentation and check if the page looks decent. 
- Ensure that changes/additions are self-documenting, i.e. that another
  developer (someone like the reviewer) will be able to understand what the code
  does in the future.

## PR checklist
